API.txt

```diff
@@ -0,0 +1,98 @@
+AC1
+POST /companies
+Request Body: {"companyID": "", "name": ""}
```

Comment:
Request body only contain name

Reply:
Good point, id shouldn't be in request body